docs/_posts/2019-10-21-megabus.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Megabus API"
+date: 2019-10-21
+excerpt: "EmoDB Megabus"
+tags: [Megabus, Kafka, EmoDB]
+type: [blog]
+---
+
+Megabus
+=======
+
+The Megabus provides a Kafka stream of ordered updates akin to the Databus, but using Apache Kafka topics to deliver updates. The API is Kafka itself. To use the Megabus, you'll discover Kafka, and then use standard Kafka clients/API's to poll and consume records from a topic.
+
+Currently, there is only one "master" Kafka topic for consumption. In the future, we will move to a model where you (1) request your own topic (with a filter for the content you care about), (2) authenticate with Kafka that you do, indeed, have access to consume this topic, and finally (3) consume your topic at the alottted rate.
+
+Discovery
+---------
+
+The Megabus Kafka topic is named based on the environment you choose. The pattern is `<universe>.emodb.<region>.megabus`.
```

Comment:
This info doesn't apply to our open source docs. Should we make a separate closed source, BV specific Confluence doc?